client/js/autocompletion.js

```diff
@@ -3,7 +3,9 @@
 const constants = require("./constants");
 
 import Mousetrap from "mousetrap";
-import {Textcomplete, Textarea} from "textcomplete";
+const {Textcomplete} = require("@textcomplete/core");
+const {TextareaEditor} = require("@textcomplete/textarea");
```

Comment:
Feels odd to mix require and import here. Does this pattern work instead?:

```js
import textcomplete from "@textcomplete/core";
const {Textcomplete} = textcomplete;
```

Reply:
yeah this will work, I just used vscodes auto import tool and didn't bother checking.